eng/Versions.props

```diff
@@ -1,11 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <MicrosoftProjectReunionInteractiveExperiencesTransportPackagePackageVersion>1.2.3-CI-25207.1001.220922-1000.1</MicrosoftProjectReunionInteractiveExperiencesTransportPackagePackageVersion>
     <MicrosoftWindowsCsWinRTPackageVersion>2.0.0</MicrosoftWindowsCsWinRTPackageVersion>
     <CsWinRTDependencyDotNetCoreSdkPackageVersion>6.0.301</CsWinRTDependencyDotNetCoreSdkPackageVersion>
     <CsWinRTDependencyDotNetCoreRuntimePackageVersion>6.0.6</CsWinRTDependencyDotNetCoreRuntimePackageVersion>
     <CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>27</CsWinRTDependencyWindowsSdkVersionSuffixPackageVersion>
     <!-- TODO: Fix DownloadDotNetCoreSdk.ps1 script so we don't need this property -->
     <CsWinRTDependencyDotNetCoreSdkLkgPackageVersion>$(CsWinRTDependencyDotNetCoreSdkPackageVersion)</CsWinRTDependencyDotNetCoreSdkLkgPackageVersion>
+
+    <!-- ShortForm Version Fields -->
+    <FrameworkUDKVersion>$(MicrosoftProjectReunionInteractiveExperiencesTransportPackagePackageVersion)</FrameworkUDKVersion>
```

Comment:
Consider removing the FrameworkUDKVersion property and use the MicrosoftProject...PackageVersion property instead.

Reply:
That field was just really long. Are you sure?

Reply:
How about IxpVersion?